### ISSUE TITLE
New u-boot revision requested for iot-gate-imx8

### ISSUE THREAD

**Issue by acostach** [balena-os/balena-iot-gate-imx8#9]
The following upstream release contains patches and updates necessary for rs232/rs485:

Version needed: U-Boot v2.3  based on NXP tag imx_v2020.04_5.4.24_2.1.0

**Comment by acostach** [comment]
Related internal discussion: https://www.flowdock.com/app/rulemotion/s-sales/threads/RuS17IlVFRNm0bWkb85yx0gBt7Y

**Comment by acostach** [comment]
Let's re-open this until the image becomes available in the dashboard

**Comment by acostach** [comment]
New OS image release 2.75.0 depends on https://github.com/balena-os/balenahup/issues/344 getting fixed, as the old dashboard OS image release 2.73.1+rev5 contains a BSP /etc/profile.d script which returns failure on term resize and HUP stops when invoked from the dashboard

**Comment by acostach** [comment]
Update: v2.80.5+rev5 will have
```
~# strings  /mnt/boot/imx-boot-iot-gate-imx8-sd.bin-flash_evk | grep U-boot -i
U-Boot SPL 2020.04-iot-gate-imx8-2.3 Balena build (May 30 2020 - 06:50:01 +0000)
Configuration to load ATF before U-Boot
U-Boot (64-bit)
U-Boot 2020.04-iot-gate-imx8-2.3 Balena build
```

**Comment by acostach** [comment]
Fixed in OS release v2.80.5+rev5
